src/include/iscsi-lio-server/widgets.rb

```diff
@@ -1445,8 +1448,24 @@ def handleClient(key, event)
 
     def validateClient(key, event)
       event = deep_copy(event)
-      ret = true
-      ret
+      clients = Convert.convert(UI.QueryWidget(:clnt_table, :Items),
+                                :from => "any",
+                                :to   => "list <term>"
+                                )
+      if clients.empty?
+        return false if !Popup.AnyQuestion(
```

Comment:
multiline trailing if is forbidden by style guide, so please use common if or better something like
```
cont = Popup.AnyQuestion( .... )
return false unless cont
```
btw order of buttons looks quite strange, as I expect that it will be positive when user want continue and not vise versa